.travis-before-script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e $DRUPAL_TI_DEBUG
```

Comment:
I *think* this file is just need when we have dependencies. can you test if it works without that?